wordpress-mobile/WordPress-iOS
# Feature/new create account

This PR updates new user/blog creation, bringing it inline with the updated sign in flow. Essentially we're porting CreateAccountAndBlogViewController to swift. 

To test: 
Be signed out and choose the Create a Site option on the sign in screen to create a new account and blog. 
- Confirm you can create a new wpcom account and blog successfully. 
- Confirm all fields are required.
- Confirm a mistyped email address is corrected by the EmailChecker.
- Confirm you can not reuse an existing email address, username, or blogname. 
- Confirm the form is completely visible on all supported device sizes, and no fields are obscured by the keyboard.
- Confirm you can successfully save newly created credentials in 1Password

@mattmiklic, this one needs a design eye.

Needs review: @koke could I trouble you for another NUX review?
